src/m365/spo/commands/eventreceiver/eventreceiver-remove.ts

```diff
@@ -0,0 +1,221 @@
+        listId: typeof args.options.listId !== 'undefined',
+        listTitle: typeof args.options.listTitle !== 'undefined',
+        listUrl: typeof args.options.listUrl !== 'undefined',
+        scope: typeof args.options.scope !== 'undefined',
+        id: typeof args.options.id !== 'undefined',
+        name: typeof args.options.name !== 'undefined',
+        confirm: (!(!args.options.confirm)).toString()
+      });
+    });
+  }
+
+  #initOptions(): void {
+    this.options.unshift(
+      {
+        option: '-u, --webUrl <webUrl>'
+      },
+      {
+        option: '--listTitle [listTitle]'
+      },
+      {
+        option: '--listId  [listId]'
+      },
+      {
+        option: '--listUrl [listUrl]'
+      },
+      {
+        option: '-n, --name [name]'
+      },
+      {
+        option: '-i, --id [id]'
+      },
+      {
+        option: '-s, --scope [scope]',
+        autocomplete: ['web', 'site']
+      },
+      {
+        option: '--confirm'
+      }
+    );
+  }
+
+  #initValidators(): void {
+    this.validators.push(
+      async (args: CommandArgs) => {
+        const isValidSharePointUrl: boolean | string = validation.isValidSharePointUrl(args.options.webUrl);
+        if (isValidSharePointUrl !== true) {
+          return isValidSharePointUrl;
+        }
+
+        const listOptions: any[] = [args.options.listId, args.options.listTitle, args.options.listUrl];
+        if (listOptions.some(item => item !== undefined) && listOptions.filter(item => item !== undefined).length > 1) {
+          return `Specify either list id or title or list url`;
+        }
+
+        if (args.options.listId && !validation.isValidGuid(args.options.listId)) {
+          return `${args.options.listId} is not a valid GUID`;
+        }
+
+        if (args.options.scope && ['web', 'site'].indexOf(args.options.scope) === -1) {
+          return `${args.options.scope} is not a valid type value. Allowed values web|site.`;
+        }
+
+        if (args.options.scope && args.options.scope === 'site' && (args.options.listId || args.options.listUrl || args.options.listTitle)) {
+          return 'Scope cannot be set to site when retrieving list event receivers.';
+        }
+
+        return true;
+      }
+    );
+  }
+
+  #initOptionSets(): void {
+    this.optionSets.push(['name', 'id']);
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      const getEventReceiversOutput = await this.getEventReceivers(args.options);
+      const eventReceivers: any = JSON.parse(getEventReceiversOutput.stdout);
+
+      if (!eventReceivers.length) { 
+        throw Error(`Specified event receiver with ${args.options.id !== undefined ? `id ${args.options.id}` : `name ${args.options.name}`} cannot be found`); 
+      }
+
+      if (eventReceivers.length > 1) { 
+        throw Error(`Multiple eventreceivers with ${args.options.id !== undefined ? `id ${args.options.id}` : `name ${args.options.name}`} found`); 
```

Comment:
Let us output the Ids to distinguish between these event receivers in a future call